testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/resource/param/HeaderDelegateTest.java

```diff
@@ -102,7 +102,7 @@ public void lastModifiedTest() throws Exception {
      */
     @Test
     public void localTest() throws Exception {
-        ResteasyProviderFactory factory = ResteasyProviderFactory.getInstance();
+        ResteasyProviderFactory factory = ResteasyProviderFactory.newInstance();

```

Comment:
Similar comment to the previous one. The test wants to check the current instance, not a new one.

Reply:
All changes complete.